Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -89,6 +89,7 @@
         'HARVESTFull',
         'HARVESTFullGlobal',
         'MiniAODFullGlobal',
+        'NanoFull',
```

Comment:
I needed this in order to get 'NanoFull' for the loop
https://github.com/cms-sw/cmssw/blob/71bc8fdf5456d1b90e1c97899bf8888a370f8259/Configuration/PyReleaseValidation/python/relval_steps.py#L3307
but I don't remember anymore if it could have any ill side effects.

Reply:
It should be fine.